utils/schemas.py

```diff
@@ -0,0 +1,133 @@
+                },
+                "refundable": {
+                    "type": "boolean",
+                    "description": "Indicates whether or not the payment can be refunded.",
+                },
+                "total_amount": {
+                    "type": "number",
+                    "description": "The amount of the payment with service fee.",
+                },
+                "service_fee": {
+                    "type": "number",
+                    "description": "The service fee amount.",
+                },
+                "amount": {
+                    "type": "number",
+                    "description": "The amount of the payment before service fee.",
+                },
+                "payment_source_channel": {
+                    "type": "string",
+                    "description": "Payment channel of the payment. Options include 'web', 'kiosk', and 'cashier'.",
+                },
+                "status": {
+                    "type": "string",
+                    "description": "The current status of the payment.",
+                },
+                "payment_type": {
+                    "type": "string",
+                    "description": "How the payment was made. Options include 'cash', 'check', 'credit_card', and 'token'.",
+                },
+                "agency": {
+                    "type": "string",
+                    "description": "The name of the agency for which the payment was made.",
+                },
+                "associated_payments": {
+                    "type": "array",
+                    "description": "An array containing other records associated with the payment. Ex. a refund record.",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "id": {
+                                "type": "integer",
+                                "description": "ID of the associated record.",
+                            },
+                            "status": {
+                                "type": "string",
+                                "description": "Status of the associated record.",
+                            },
+                        },
+                    },
+                },
+                "created_at": {
+                    "type": "string",
+                    "description": "When the payment was made. Timestamps are in UTC in ISO-8601 format.",
+                },
+                "line_items": {
+                    "type": "array",
+                    "description": "An Array of Line Items.",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "id": {
+                                "type": "string",
+                                "description": "The ID of the Line Item. Ex. The ID of a Permit being paid for.",
```

Comment:
This description took me a second to recognize that the Ex. meant example to follow. Maybe something like this for this description? I'm not sure -- yours is plenty fine too.

"Line Item ID: The unique identifier of the item being processed (e.g., a permit id)"